src/clients/slaos.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+import httpx
+import stamina
+from pydantic import BaseModel
+
+from src.config.models.output import RatedOutputConfig
+
+
+class SlaosClient:
+    def __init__(self, config: RatedOutputConfig):
```

Comment:
not sure if it should be a class - it does have one method and no state -  creating two functions: `get_latest_ingest_timestamp(ingest_url, datastream_key)` and `make_ingest_url(*, base_url, id, key)` might be just simpler